_includes/footer.html

```diff
@@ -22,7 +22,8 @@
                             <ul class="contact-methods">
                                 <li><i class="icon pe-7s-mail"></i><span><a id="mailAdd" href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
                                 <li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-                                <li><i class="icon pe-7s-map-marker"></i><span>FOSSASIA</span></li>
+                                        <li><a href="https://www.google.co.in/maps/search/6+Eu+Tong+Sen+Street+%2305-07+The+Central,+Singapore+059817/@1.2889009,103.8443213,17z/data=!3m1!4b1?hl=en">
```

Comment:
Hi @samyakbvs No one going to click on the map_marker icon. You can add it to the address given

Reply:
okay no problem